code/index.html

```diff
@@ -1,30 +1,158 @@
+        <div class="form-group">
+      </form>
+
+      <form class="form" action="http://httpbin.org/anything" method="POST">
+
+<div class="messagearea">
+      <label for="message">Write to us <p>(max 90 characters)</p></label>
+      <textarea class="w3-input" id="message" cols="10" rows="2" placeholder="Type your message" name="messageValue"
+        minlength="4" maxlength="90" required></textarea>
+      </div><br>
+      <button class="form-submit-two" type="submit"></button>
+      </form>
+    </div>
+    </section>
+
+
+
+    <!-- fmsdfkm  -->
+
+    <section class="content-item2">
+
+      <h1>About</h1>
+
+      <div class="wrapper"></div>
+
+      <button class="accordion" typ="button">Lorem </button>
+      <div class="panel-a">
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+          industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+          scrambled it to make a type specimen book.</p>
       </div>
+      <button class="accordion" typ="button">Lorem</button>
+      <div class="panel-b">
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+          industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+          scrambled it to make a type specimen book.</p>
+      </div>
+      <button class="accordion" typ="button">Lorem</button>
+      <div class="panel-a">
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+          industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+          scrambled it to make a type specimen book. </p>
+      </div>
+      <button class="accordion" typ="button">Lorem</button>
+      <div class="panel-b">
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+          industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+          scrambled it to make a type specimen book. </p>
+      </div>
+      <button class="accordion" typ="button">Lorem</button>
+      <div class="panel-a">
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+          industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+          scrambled it to make a type specimen book. </p>
+      </div>
+      </div>
+
     </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+  </section>
+
+
+
+  <div class="footer">
+
+    <video autoplay loop muted class="footer__video">
+      <source src="Sky - 3186.mp4">
+    </video>
+    <div class="footer_overlay">
+<h1>Nathalie Nordström</h1>
+    </div>
```

Comment:
This isn't visual on your site, so either remove it or let it show on your website. Also think about how to use the heading, the `<h1>`should be used on the biggest text on the site.

Reply:
I took it away